native/c/zowex.cpp

```diff
@@ -543,10 +543,10 @@ int handle_job_list(ZCLIResult result)
 {
   int rc = 0;
   ZJB zjb = {0};
-  string owner_name(result.get_option("--owner").get_value());
-  string prefix_name(result.get_option("--prefix").get_value());
-  string max_entries = result.get_option("--max-entries").get_value();
-  string warn = result.get_option("--warn").get_value();
+  string owner_name(result.get_option("--owner")->get_value());
+  string prefix_name(result.get_option("--prefix")->get_value());
+  string max_entries = result.get_option("--max-entries")->get_value();
+  string warn = result.get_option("--warn")->get_value();
```

Comment:
i think these might cause access violations at runtime if we don't check that option is not `nullptr` first